REACT-TODO-22ND/src/styles/theme.ts

```diff
@@ -0,0 +1,28 @@
+export const theme = {
+  colors: {
+    primary: 'green',
+    primaryDark: 'darkgreen',
+    white: '#ffffff',
+    lightGray: '#f8f9fa',
+    gray: '#e9ecef',
+    darkGray: '#6c757d',
+    lightBlue: '#e3f2fd',
+    yellow: '#fff3cd',
+    yellowDark: '#ffeaa7',
```

Comment:
사소하지만 일관성을 위해 위 color들과 동일한 순서로 네이밍해 주시면 좋을 것 같습니당!
```suggestion
    darkYellow: '#ffeaa7',
```